### PR TITLE
Xcode 10 / Swift4.2 support

### PR DESCRIPTION
**Pull request by intere**
The original project didn't run under Xcode 10, so I updated it so that it does.

### CODE REVIEW

File: AppleTVRemoteDemoApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -90,6 +90,7 @@
 				TargetAttributes = {
 					4FCB9BC41BCC608D00B4F841 = {
 						CreatedOnToolsVersion = 7.1;
+						DevelopmentTeam = 268S5GJFWZ;
```
Comment (lucianomarisi) [inline AppleTVRemoteDemoApp.xcodeproj/project.pbxproj]: Is this project change needed?

File: AppleTVRemoteDemoApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -230,20 +231,24 @@
 		4FCB9BD51BCC608E00B4F841 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				DEVELOPMENT_TEAM = 268S5GJFWZ;
 				INFOPLIST_FILE = AppleTVRemoteDemoApp/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.TechBrewers-LTD.AppleTVRemoteDemoApp";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 4.2;
```
Comment (lucianomarisi) [inline AppleTVRemoteDemoApp.xcodeproj/project.pbxproj:238]: In Xcode 10 wouldn't this be the default?

**Comment by lucianomarisi** [comment]
Would it be possible to update the travis file so that CI passes?

File: AppleTVRemoteDemoApp.xcodeproj/project.pbxproj (hunk 1 above)
Comment (intere) [reply AppleTVRemoteDemoApp.xcodeproj/project.pbxproj]: @lucianomarisi - no it's not, I'll revert that, I was testing on my TV device.

File: AppleTVRemoteDemoApp.xcodeproj/project.pbxproj (hunk 2 above)
Comment (intere) [reply AppleTVRemoteDemoApp.xcodeproj/project.pbxproj:238]: If you create a new project, yes it's the default.  For an existing project, however; you have a choice (the value was initially "Unspecified"). 

**Reference Screenshot**

image

**Comment by intere** [comment]
@lucianomarisi - PR is updated, CI is back to normal.

**Comment by lucianomarisi** [comment]
Nice, thanks a lot for this PR!

**Pull request merged**